src/main/java/jenkins/plugins/git/traits/FirstBuildChangelogTrait.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2017 CloudBees, Inc.

```

Comment:
Copyright date should be this year.

Also, check to see if copyright dates may need to be updated in other files modified.

Reply:
Copyright should not be assigned to CloudBees usually. Author usually retains copyright.

Reply:
I have removed the CloudBees copyright documentation, would an author descriptor be adequate for this file?

Reply:
Either is fine. If you want to declare copyright explicitly, that is fine. If you don't list it explicitly, the creation of the file makes it copyright by you implicitly. 

Thanks for your contribution